syft/tensor.py

```diff
@@ -1372,6 +1374,30 @@ def mm(self, tensor2):
 
         return syft.mm(self, tensor2)
 
+    def stride(self, dim=None):
+        """Returns the jump necessary to go from one element to the next one in the specified dimension dim."""
```

Comment:
Please make the documentation consistent with #252

you can take a look at this link 
https://github.com/OpenMined/PySyft/blob/4e2aa9e34df985f4d8e0703d900eaa2558fd413a/syft/math.py#L259